app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    unvaccinated_friends_count = 0
+    lacking_masks_count = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            lacking_masks_count += 1
+        except VaccineError:
+            unvaccinated_friends_count += 1
+
+    if unvaccinated_friends_count != 0:
+        return "All friends should be vaccinated"
```

Comment:
If at least one of the friend has problems with vaccines the function should return a message `"All friends should be vaccinated".
you don't need to count all of them